src/qt/bitcoingui.cpp

```diff
@@ -351,6 +354,11 @@ void BitcoinGUI::createActions()
     // initially disable the debug window menu item
     openRPCConsoleAction->setEnabled(false);
 
+    showMempoolStatsAction = new QAction(platformStyle->TextColorIcon(":/icons/mempoolstats"), tr("&Mempool Statistics"), this);
```

Comment:
Huh, I recall he gave you permission to dual-license as MIT?

dae0a89d4b66f08c83ccc8c20cf37521084b6257